cluster-stamp.json

```diff
@@ -821,7 +821,8 @@
                         "name": "npsystem",
                         "count": 3,
                         "vmSize": "Standard_DS2_v2",
-                        "osDiskSizeGB": 512,
+                        "osDiskSizeGB": 80,
```

Comment:
1) 512 GiB was chosen to support proper IOPS (when using Managed disks)
1) Moving to Ephemeral the IOPS constraint is gone
1) In order to keep 512GB, we'd need to bump the SKU to the largest SKU in the series.
1) The system node technically doesn't need more than 30 GiB, 80 GiB was seen as a safe selection (max on this SKU is 86 GiB)

Reply:
You'll noticed I moved the user node pool to 110 since we have a bit more space there and I'd rather support larger images OOTB there so customers don't run into space issues too quickly :)